src/main/webapp/WEB-INF/classes/spring/fcrepo4-config.xml

```diff
@@ -0,0 +1,232 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Sorry about that, clearly its going to take me some time to avoid that abbreviation.